ow_sim_tests/test/arm_check_action.py

```diff
@@ -0,0 +1,294 @@
+    if expected_final is not None:
+
+      self._assert_point_is_near(
+        result.final, expected_final, expected_final_tolerance,
+        "Arm did not complete the %s action in the position expected! Fianl position is x =%s  y =%s  z =%s" 
+          % (action_name, result.final.x, result.final.y, result.final.z)
+      )
+      
+
+    return result
+
+  def test_01_unstow(self):
+    UNSTOW_MAX_DURATION = 30.0
+    unstow_result = self._test_action(
+      'Unstow',
+      ow_lander.msg.UnstowAction,
+      ow_lander.msg.UnstowGoal(),
+      UNSTOW_MAX_DURATION,
+      self._assert_nothing,
+      expected_final = Point(1.7419, 0.2396, -6.5904),
+      server_timeout = 25.0
+    )
+
+  def test_02_guarded_move(self):
+    GUARDED_MAX_DURATION = 60.0
+    guarded_move_result = self._test_action(
+      'GuardedMove',
+      ow_lander.msg.GuardedMoveAction,
+      ow_lander.msg.GuardedMoveGoal(),
+      GUARDED_MAX_DURATION,
+      self._assert_nothing,
+      expected_final =  Point(2.0324, -0.1012, -0.1585),
+      start = Point(2.0, 0.0, 0.3),
+      normal = Point(0.0, 0.0, 1.0),
+      search_distance = 0.5
+    )
+
+  def test_03_unstow(self):
+    UNSTOW_MAX_DURATION = 30.0
+    unstow_result = self._test_action(
+      'Unstow',
+      ow_lander.msg.UnstowAction,
+      ow_lander.msg.UnstowGoal(),
+      UNSTOW_MAX_DURATION,
+      self._assert_nothing,
+      expected_final = Point(1.7419, 0.2396, -6.5904),
+      server_timeout = 25.0
+    )
+
+  def test_04_grind(self):
+    GRIND_MAX_DURATION = 80.0 
+    grind_result = self._test_action(
+      'Grind',
+      ow_lander.msg.GrindAction,
+      ow_lander.msg.GrindGoal(),
+      GRIND_MAX_DURATION,
+      self._assert_nothing,
+      expected_final = Point(1.4720, -0.1407, -6.7400),
+      x_start         = 1.65,
+      y_start         = 0.0,
+      depth           = 0.15,
+      length          = 0.7,
+      parallel        = True,
+      ground_position = -0.155
+    )
+
+  def test_05_dig_circular(self):
+    DIG_CIRCULAR_MAX_DURATION = 60.0
+    dig_circular_result = self._test_action(
+      'DigCircular',
+      ow_lander.msg.DigCircularAction,
+      ow_lander.msg.DigCircularGoal(),
+      DIG_CIRCULAR_MAX_DURATION,
+      self._assert_nothing,
+      expected_final = Point(1.6499, -0.1219, -7.3220),
+      x_start         = 1.65,
+      y_start         = 0.0,
+      depth           = 0.01,
+      parallel        = False,
+      ground_position = -0.155
+    )
+
+  def test_06_grind(self):
+    GRIND_MAX_DURATION = 80.0 
+    grind_result = self._test_action(
+      'Grind',
+      ow_lander.msg.GrindAction,
+      ow_lander.msg.GrindGoal(),
+      GRIND_MAX_DURATION,
+      self._assert_nothing,
+      expected_final = Point(1.4720, -0.1407, -6.7400),
+      x_start         = 1.65,
+      y_start         = 0.0,
+      depth           = 0.15,
+      length          = 0.7,
+      parallel        = True,
+      ground_position = -0.155
```

Comment:
I tried using those arguments, it failed the grind test. So I keep the old defaults.

Reply:
It does not necessarily need to match the parameters in `arm_check_service.py` because this test really only ensures execution of each arm operation occurs. The current parameters being used now work fine, so lets keep it that way.